src/components/Feed/Feed.css

```diff
@@ -0,0 +1,15 @@
+.articles-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
```

Comment:
The article container is set to flex-row to keep the feed centered on the page.
This means that flex-column can be applied to the articles display while
keeping the row-centered orientation.

React natively has some center orientation applied, but it can cause issues when
orienting individual elements. Applying flex-center to a container overrides the
native centering in the cascade and makes it easier to troubleshoot styling
issues.

Reply:
I think we might need to add a CSS property for the article-image class if we are going to go the route of phone application, as when I set my dev tools to iPhone 6/7/8 or Galaxy S5, the article images start to overflow the articles-container.

Reply:
I think we might need to add a CSS property for the article-image class if we are going to go the route of phone application, as when I set my dev tools to iPhone 6/7/8 or Galaxy S5, the article images start to overflow the articles-container.
![Screen Shot 2021-11-01 at 11 29 04 AM](https://user-images.githubusercontent.com/68795578/139714086-10a336fe-2047-477d-8a7f-85ced898c625.png)

Reply:
Got it! I'll start adding media queries to resolve this during my refactor. I noticed that happening this morning, too.